build.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Current azure pipelines [docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-yaml?view=azure-devops)/[examples](https://github.com/MicrosoftDocs/pipelines-javascript) appear to recommend a canonical filename of `azure-pipelines.yml` rather than `build.yml`; let's be consistent with the recommendation in the new repo

Reply:
build.yaml is much more helpful/general than `azure-pipelines.yml` and since we have already been using it. I think it doesn't hurt us to keep using `build.yaml`. 

Its also something of a normal thing for a non-familiar user to look for in a repo. So, I am more inclined towards `build.yaml`

Reply:
Can you help me understand *why* you think it's more helpful?

In terms of helpfulness for a non-familiar user, I think searchability of the file name is the most important thing. "build.yaml" and "azure-pipelines.yml" are equally searchable (both have first results pointing to the right docs), and if you try without the extension, "azure pipelines" is much more searchable than "build".

I agree that it is much more general, but I think that makes the name less helpful, not more helpful.

On "since we have already been using it": I agree that build.yaml is more consistent with accessibility-insights-web, but I think it's less consistent with the official docs, and I think consistency with the official docs is more important than consistency with our other tangentially related project.

Reply:
Its more of a thing of preference where we want to align ourselves. If we want to be consistent in the projects that we already have. I remember having some discussion around this when we introduced `build.yaml` in the `ai4web` repo and every one was kind of in favor of `build.yaml` over `azure-pipelines.yml`. But, if we want to follow the `azure-pipelines.yml` strategy, I dont mind and will change it.